examples/test/BUILD.bazel

```diff
@@ -0,0 +1,9 @@
+cc_test(
+    name = "simple_test",
+    srcs = ["simple_test.cpp"], 
```

Comment:
I was thinking of following the Google style of coding for our codebase:
Here is the style guide: [link](https://google.github.io/styleguide/cppguide.html#Header_Files).

So, based on this change the name of this to `simple_test.cc`.

Reply:
hmm, I don't get the point behind using .cc extension, this seems to be uncommon extension used mostly in google's code bases. Apart from being mentioned in the Style guide are there any advantages of following this?